docs/asciidoc/driver-configuration.adoc

```diff
@@ -0,0 +1,187 @@
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+
+const server = new ApolloServer({
+    schema: neoSchema.schema,
+    context: ({ req }) => ({ req }),
+});
+----
+
+==== Driver in context
+
+[source, javascript]
+----
+const { Neo4jGraphQL } = require("@neo4j/graphql");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+
+const server = new ApolloServer({
+    schema: neoSchema.schema,
+    context: ({ req }) => ({ req, driver }),
+});
+----
+
+=== OGM
+
+[source, javascript]
+----
+const { OGM } = require("@neo4j/graphql-ogm");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const ogm = new OGM({ typeDefs, driver });
+----
+
+[[driver-configuration-database-compatibility]]
+== Database Compatibility
+
+Use the `checkNeo4jCompat` method available on either a `Neo4jGraphQL` or `OGM` instance to ensure the specified DBMS is of the required version, and has the necessary functions and procedures available. The `checkNeo4jCompat` will throw an `Error` if the DBMS is incompatible, with details of the incompatibilities.
+
+=== `Neo4jGraphQL`
+
+[source, javascript]
+----
+const { Neo4jGraphQL } = require("@neo4j/graphql");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+await neoSchema.checkNeo4jCompat();
+----
+
+=== `OGM`
+
+[source, javascript]
+----
+const { OGM } = require("@neo4j/graphql-ogm");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const ogm = new OGM({ typeDefs, driver });
+await ogm.checkNeo4jCompat();
+----
+
+== Specifying Neo4j database
+
+The Neo4j database may be added to the GraphQL context object:
+
+[source, javascript]
+----
+const { Neo4jGraphQL } = require("@neo4j/graphql");
+const neo4j = require("neo4j-driver");
+
+const typeDefs = `
+    type User {
+        name: String
+    }
+`;
+
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
+
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
+
+const server = new ApolloServer({
+  schema,
+  context: { driverConfig: { database: "my-database" } }
+});
+----
+
+== Specifying Neo4j Bookmarks
```

Comment:
It might be worth noting that you can also specify the database and bookmarks in the Neo4jGraphQL constructor.